ConfigurationValidation.hpp

```diff
@@ -34,6 +34,20 @@
     #error Unsupported display configuration. Use at own risk.
 #endif
 
+// Info Display validations
+#if defined(__AVR_ATmega2560__)
+    #if (INFO_DISPLAY_TYPE != INFO_DISPLAY_TYPE_NONE)
+        #if (DISPLAY_TYPE != DISPLAY_TYPE_NONE)
+            #error Unsupported configuration. Not enough memory to use LCD and OLED at the same time.
+        #endif
+        #if (FOCUS_STEPPER_TYPE != STEPPER_TYPE_NONE)
+            #if (BOARD == BOARD_AVR_MKS_GEN_L_V1) || (BOARD == BOARD_AVR_MKS_GEN_L_V2) || (BOARD == BOARD_AVR_MKS_GEN_L_V21)
+                #error Unsupported configuration. OLED requires I2C, which is on the E1 port on a MKS GenL, so a focuser cannot be used
```

Comment:
The focuser doesn't need to be on E1, maybe we could check the pins? But we can only go from stepper to pin, not check if a pin is used anywhere... (afaik). :frowning_face: 

We should err on the side of not producing a false positive though, since it could be a valid configuration that we're blocking.

Reply:
Ahh, yes, I see. You could put the focuser on another port if you're not using ALT AZ.... Should I change it to a warning? Should we generally just emit a warning to not use E1 when this is enabled?

Reply:
Changed to warning.